taxonomy/serializers.py

```diff
@@ -18,4 +48,20 @@ class Meta:
         """
 
         model = Skill
-        fields = ('name', 'description')
+        fields = ('name', 'description',)
+
+
+class SkillDetailsSerializer(SkillSerializer):
```

Comment:
I think it would be better to add category and subcategory fields inside the existing `SkillSerializer` instead of adding a new serialiser.

Reply:
Right, I actually did the prior.  I was under the impression that ES document would get affected by it. But I was checking ES course doc https://github.com/openedx/course-discovery/blob/master/course_discovery/apps/course_metadata/search_indexes/documents/course.py and it seems Skill's name and description are added there only. I will update this.